DESCRIPTION

```diff
@@ -51,7 +51,8 @@ License: MIT + file LICENSE
 Encoding: UTF-8
 LazyData: true
 Imports: 
-    checkmate, 
+    checkmate,
+    cli,
```

Comment:
I'm seeing this appear in a few different places in the imports list so we will need to keep a careful eye that it doesn't slip in twice.